src/aspire/utils/blk_diag_matrix.py

```diff
@@ -48,18 +51,16 @@ def __setitem__(self, key, value):
         self.data[key] = value
 
     def __len__(self):
-        #return self.data.__len__
         return self.nblocks
 
-
     @staticmethod
     def zeros(blk_partition, dtype=np.float64):
         """
         Build a BlkDiagMatrix zeros matrix
 
         :param blk_partition: The matrix block partition in the form of a
         K-element list storing all shapes of K diagonal matrix blocks,
-        where `blk_partition[i]` corresponds to the shape (number of rows and columns)
+        where `blk_partition[i]` corresponds to the shape (number of rows cols)
```

Comment:
Numbers of rows and cols?

Reply:
(i'll fix all of them together, you don't need to make an item for each one, save your time :) )